src/test/ui/error-codes/E0697.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0697]: closures cannot be static
+  --> $DIR/E0697.rs:2:5
+   |
+LL |     static || {};
+   |     ^^^^^^^^^ help: consider removing the: `static`
```

Comment:
This suggestion won't apply correctly, because the resulting file will be `static {};`, which is not what we want. Also, I would change the wording to be more prescriptive: removing `static` isn't optional here, you just can't do that.

Reply:
Oh, thought you could. Should I just remove this change?

Reply:
You can change this to be a `span_label` instead.

Reply:
Wait, your code changes do what we want. Disregard.

Reply:
Oh